doc/json-rpc.md

```diff
@@ -227,3 +234,115 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "m
 ```
 
 ---
+
+### subscriptionChannel
+Open a socket on the node for JSON-RPC subscription requests
+
+##### Parameters:
+* `integer` - port to open for websocket subscription requests
+
+##### Results:
+* `string` - "addr", open websocket address
```

Comment:
I'd just return the port here and not the full address that the node thinks it has (there might be a nat in between..)